packages/components/src/components/NFT/NFT.tsx

```diff
@@ -47,7 +46,9 @@ export const NFT = ({ contractAddress, tokenId, size = 'xs' }: NFTProps) => {
 
   const fetchNFTData = useCallback(async () => {
     try {
-      const res = await fetch(`https://api.opensea.io/api/v1/asset/${contractAddress}/${tokenId}/`);
+      const res = await fetch(
```

Comment:
Most of these formatting changes are due to the lint-staged not actually applying correctly before it seems